src/components/Authentication/Login.vue

```diff
@@ -3,7 +3,26 @@
     <div class="card-body">
       <img src="../../assets/logo.png" class="rounded mx-auto d-block img-fluid" id="falkoLogoLogin">
 
-      <form id="loginForm"  @submit.prevent="login()" name="wrong-credentials">
+      <div v-if="this.alert" class="alert alert-danger fade show" role="alert">
+        <div class="row">
+          <div class="column alert-column-left">
+            Incorrect email or password
```

Comment:
It would be great if this message was the returned by the API. So the message would be dynamic. Can you take at look to see if this is possible?

Reply:
Ok, but I should change the message from the back-end to "Incorrect email or password" or send the message which is already being sent to the front-end: "invalid credentials" ?